inc/saitypes.h

```diff
@@ -944,6 +944,88 @@ typedef struct _sai_port_eye_values_list_t
     sai_port_lane_eye_values_t *list;
 } sai_port_eye_values_list_t;
 
+/**
+ * @brief System port configuration attributes
+ */
+typedef struct _sai_system_port_config_t
+{
+    uint32_t port_id;
+    uint32_t attached_switch_id;
+    uint32_t attached_core_index;
```

Comment:
please add some description for each field and also what are mandatory argument that basic VOQ systemport needs.  If I remember correctly speed was not mandatory.

Reply:
Will update the comments.

Reply:
Will add comments. Only optional member is num_voq. Speed is needed for rate class scheduling on VOQ.

Reply:
Moved mandatory comment to top level data structure comment